Source/UnitTests/Core/IOS/ES/TestBinaryData.h

```diff
@@ -8,8 +8,170 @@
 #include <cstddef>
 #include "Common/CommonTypes.h"
 
-extern const std::array<u8, 520> soup01_tmd;
-extern const std::array<u8, 108> soup01_tmd_view;
+constexpr std::array<u8, 520> soup01_tmd = {{
```

Comment:
Why was this data moved to the header?

Reply:
See the corresponding commit name. I couldn't get it to compile otherwise.

Reply:
Might need /Zc:externConstexpr or constexpr -> const in the cpp file.